src/GeositeFramework/plugins/launchpad/style.css

```diff
@@ -26,3 +26,41 @@
     list-style-type: none;
     padding: 0;
 }
+
+.launchpad-plugin .plugins .button.truncate-toggle, .scenarios .button.truncate-toggle {
+    padding: 2px !important;
+    font-size: 0.7em !important;
+    width: 35%;
+    float: none;
+}
+
+.truncate-box {
+    overflow: hidden;
+    position: relative;
+    line-height: 1.2em;
+    max-height: 2.6em;
+    text-align: justify;
+    margin-right:-1em;
+    padding-right: 2em;
+}
+
+.truncate-container > p {
+    width: 90%;
```

Comment:
The text width is much narrower than the original design. I think some of this CSS and markup changes should be changed to preserve the original style.

Before:
![image](https://cloud.githubusercontent.com/assets/43062/21296958/13022186-c545-11e6-924f-f6b23704b2c5.png)

After:
![image](https://cloud.githubusercontent.com/assets/43062/21296946/eee8cb60-c544-11e6-8486-805ed02f8292.png)